contrib/inventory/foreman.py

```diff
@@ -22,6 +22,7 @@
 # Stdlib imports
 # __future__ imports must occur at the beginning of file
 from __future__ import print_function
```

Comment:
Could you do this on the same line?
```suggestion
from __future__ import absolute_import, print_function
```